design/apps.go

```diff
@@ -0,0 +1,278 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+// SimpleSpace describe a space
+var simpleSpace = a.Type("SimpleSpace", func() {
```

Comment:
the responses do not comply with the JSON-API spec. You should have `ID` and `Type` as required fields, then an `Attributes` field that contains all other attributes of the resource. See https://github.com/fabric8-services/fabric8-tenant/blob/master/design/tenant.go#L8-L18 for examle

Reply:
It's completely true. we weren't aiming for compliance because this was thought of in terms of an API for use by the deployments page only.  (Also, inexperience with the JSON-API spec)
I am very happy to make it compliant, but I think it should be a separate set of PRs, since it will require changes on the UI side too.

Reply:
oh, I did not realize that you already have a UI working with this PR. Ok then, but this change remains to be done.

Reply:
I've already coded it in my own repo; do you mean it must be part of the PR, or can it be a follow up (so I can properly coordinate it with the UI changes)?

Reply:
yes, in a follow-up PR it's fine (this one is already quite big anyways)